src/internal.c

```diff
@@ -8495,6 +8495,12 @@ static int DoVerifyCallback(WOLFSSL* ssl, int ret, ProcPeerCertArgs* args)
             WOLFSSL_MSG("Verify callback overriding error!");
             ret = 0;
         }
+        else {
```

Comment:
We need test cases for:

1) no error going into callback, return ok
2) no error going into callback, return error
3) error going into callback, return ok
4) error going into callback, return error

Assign to someone else for the tests if needed.

Reply:
It's not clear to me that all 4 of these cases are being tested.  I think there are new cases for 4) though I don't see it clearly documented in terms of the callback entrance state.  We already had 3) with the -j option.

Where are cases 1 and 2)?  Let's add better documentation if they're there.